broker/api.py

```diff
@@ -294,18 +294,17 @@ def update(  # noqa C901 # TODO: simplify this function
         if instance.has_active_operations():
             raise errors.ErrBadRequest("Instance has an active operation in progress")
 
-        (updated_domain_names, no_domain_updates) = handle_domain_updates(
-            params, instance
-        )
+        updated_domain_names = handle_domain_updates(params, instance)
```

Comment:
tiny nit - to me `updated_domain_names` reads like a delta, or a list of the names that have changes. Maybe `new_domain_names` or `requested_domain_names`?